static/js/controllers/contacts-content.js

```diff
@@ -119,26 +120,57 @@ var _ = require('underscore');
         var split = _.partition(children, function(child) {
```

Comment:
Hmm. I want to pull the contact to the top, but also leave the other elements in the same order, which `sort` might not do. So no.
 
>"If compareFunction(a, b) returns 0, leave a and b unchanged with respect to each other, but sorted with respect to all different elements. Note: the ECMAscript standard does not guarantee this behaviour, and thus not all browsers (e.g. Mozilla versions dating back to at least 2003) respect this."